arangod/Agency/State.cpp

```diff
@@ -1188,20 +1188,11 @@ bool State::loadRemaining(index_t cind) {
                                             : std::string();
 
           uint64_t millis = 0;
-          if (ii.hasKey("epoch_millis")) {
-            if (ii.get("epoch_millis").isInteger()) {
-              try {
-                millis = ii.get("epoch_millis").getNumber<uint64_t>();
-              } catch (std::exception const& e) {
-                LOG_TOPIC("2ee75", FATAL, Logger::AGENCY)
-                  << "Failed to parse integer value for epoch_millis " << e.what();
-                FATAL_ERROR_EXIT();
-              }
-            } else {
-              LOG_TOPIC("52ee7", FATAL, Logger::AGENCY) << "epoch_millis is not an integer type";
-              FATAL_ERROR_EXIT();
-            }
-          }
+	  if (auto milliSlice = ii.get("epoch_millis"); milliSlice.isNumber()) {
```

Comment:
No objection against the code change here, but the indention level of the added code looks wrong.

Reply:
Just checked. looks fine

Reply:
Because you used a tabstop to indent. But we don't use tab stops to indent code.